make_obsparams.py

```diff
@@ -90,6 +133,14 @@ def cli():
 # TODO: These options are the same as options in `run_sim.py`. We can define them as
 # a variable in utils and do @variable.
 @cli.command("h4c")
```

Comment:
Change this decorator to `@click.command()`

Reply:
Cool.